src/test/java/io/crate/jmx/integrationtests/V560MetricsITest.java

```diff
@@ -22,12 +22,24 @@
 
 package io.crate.jmx.integrationtests;
 
-public class V430MetricsITest extends MetricsITest {
+import org.junit.Test;
 
-    private static final String URL = "https://cdn.crate.io/downloads/releases/crate-4.3.0.tar.gz";
+public class V560MetricsITest extends MetricsITest {
```

Comment:
Again nothing changed for `5.4`, `5.5`, `5.6`. I'd remove...

Reply:
Ok, I will just keep `4.8` and `5.7` + the MetricItest which tests latest nightly.

Reply:
I only kept 5.7, 4.8 supports the same metrics, so...